spec.html

```diff
@@ -12165,7 +12165,7 @@ <h1>Runtime Semantics: BindingInitialization</h1>
         <h1>Runtime Semantics: InitializeBoundName ( _name_, _value_, _environment_ )</h1>
         <p>The abstract operation InitializeBoundName takes arguments _name_, _value_, and _environment_. It performs the following steps when called:</p>
         <emu-alg>
-          1. Assert: Type(_name_) is String.
+          1. Assert: Either Type(_name_) is String, or _name_ is ~default~.
           1. If _environment_ is not *undefined*, then
             1. Perform _environment_.InitializeBinding(_name_, _value_).
```

Comment:
I am pretty sure we shouldn't actually be attempting to initialize a binding named `~default~`.

Reply:
fixed in 870e5f7cc27ac0e7489059d3c650595b100a2796